engines/director/lingo/lingo-the.cpp

```diff
@@ -260,6 +262,7 @@ TheEntityField fields[] = {
 	{ kTheField,	"textHeight",	kTheTextHeight,	300 },//		D3 p
 	{ kTheField,	"textSize",		kTheTextSize,	300 },//		D3 p
 	{ kTheField,	"textStyle",	kTheTextStyle,	300 },//		D3 p
+	{ kTheField,	"scrollTop",	kTheScrollTop,  300 },//		D3 p
```

Comment:
Which Director version did you add it for? It is not present in D3 or D4. Was it D5?